<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="addieren">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>51</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string>Addieren</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="eingabezahl1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="eingabezahl2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>51</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="ergebnisausgabe">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>380</y>
     <width>191</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="zahl1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Zahl 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="zahl2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Zahl 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="zahl3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>390</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>Ergebnis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="subtrahieren">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtrahieren</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiplizieren">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Multiplizieren</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dividieren">
   <property name="geometry">
    <rect>
     <x>689</x>
     <y>320</y>
     <width>81</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>Dividieren</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
